alacritty_terminal/src/term/mod.rs

```diff
@@ -815,14 +815,17 @@ pub struct Term<T> {
     input_needs_wrap: bool,
 
     /// Alternate grid.
-    alt_grid: Grid<Cell>,
+    inactive_grid: Grid<Cell>,
 
     /// Alt is active.
     alt: bool,
```

Comment:
Is there a reason why this hasn't been removed yet?

Reply:
I’ve been busy, I only really have time to look at this on the weekend. I tried removing this in the first iterations of the PR and ran into test failures.

There are some things I think I can do to remove this and make things work